### PR TITLE
Sandbox Postgres support with passing api integration tests

### PR DESCRIPTION
**Pull request by gaboraranyossy-da**
### Pull Request Checklist

- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/SandboxSubmissionService.scala [hunk 1]
```diff
@@ -92,7 +93,13 @@ class SandboxSubmissionService private (
         recordOnLedger(commands).transform {
           case Failure(error) =>
             logger.warn(s"Submission of command ${commands.commandId.unwrap} has failed.", error)
-            Failure(error)
+            // quick work-around to handle back-pressure coming from Akka
+            //TODO: we should encapsulate the back pressure signal in a proper type and make recordOnLedger return a Future[OK \/ BackPressure]
```
Comment (francesco-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/SandboxSubmissionService.scala]: can you do this as part of this pr? that feels very hacky....

**Comment by gaboraranyossy-da** [comment]
@neil-da it's labeled as WIP and as such it's not ready for review yet. I merely wanted to give our CI a spin

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/SandboxSubmissionService.scala (hunk 1 above)
Comment (gaboraranyossy-da) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/SandboxSubmissionService.scala]: yes, I can. I will literally do the same just in a lower layer.

**Comment by neil-da** [comment]
@gaboraranyossy the purpose of a description is for human readers. Whether the code is draft or not it still ends up in my inbox, and I still want to understand that I don't need to read it :)

**Comment by gaboraranyossy-da** [comment]
which is why we have the WIP label :)

**Comment by francesco-da** [comment]
let's re-open this as draft pr

**Pull request closed without merging**